### ISSUE TITLE
Support Subset of the lunarvim IDE using a python script.

### ISSUE THREAD

**Issue by dariusgm** [dariusgm/lunarvim#8]

**Comment by damn-war** [comment]
Are you thinking about something like a frontend (not necessarily a GUI, a TUI is fine, too), where a user
- can choose subsets of the IDE
- can choose which files/folder he/she wants to mount (maybe with suggestions for ssh-config or so)
- can add an .env file 
- ...
?

**Comment by dariusgm** [comment]
Yes, that was the idea. Still to have the different languages better separated, the structure needs to be changes - for examples adding the language support not all in one place (the main Dockerfile in the repository) but have it more close connected to the language you want to use.
